shell/public/pwa-service-worker.js

```diff
@@ -0,0 +1,18 @@
+
+self.addEventListener('install', function(event) {
+	const urlsToCache = [
+		// TODO: we should add static assets and such to this list.
+	]
+	event.waitUntil(caches.open('sandstorm-cache').then(function(cache) {
+		return cache.addAll(urlsToCache);
+	}))
+	self.addEventListener('fetch', function(event) {
+		event.respondWith(
+			caches.match(event.request).then(function(response) {
```

Comment:
Given that we aren't actually caching anything, should we remove the `caches.match` check here? It seems like it'll just make things slower.

Does it work if you don't register a `fetch` listener at all? (I think it should then fall back to the default behavior of sending the request to the server...)

Reply:
Ack, I have another commit locally that actually does cache static assets, apparently I forgot to push... sorry about the slopyness on this one.